src/drivers/gps/gps.cpp

```diff
@@ -1560,4 +1569,4 @@ int
 gps_main(int argc, char *argv[])
 {
 	return GPS::main(argc, argv);
-}
\ No newline at end of file
+}
```

Comment:
I don't think this is needed

Reply:
The PX4 file ends with a blank line.
Some compilers require a blank line.